update-since-todo.sh

```diff
@@ -25,7 +25,7 @@ do
         echo -e "\tfirst tag was $firstTag"
         echo -e "\tUpdating file in place"
         sedExpr="${line}s/@since TODO/@since ${firstTag//jenkins-/}/"
-        sed -i $sedExpr $file
+        sed -i.bak $sedExpr $file
```

Comment:
Can’t you do an empty string instead of .bak? I.e ‘’ (on mobile so can’t format properly)

Reply:
The problem is that…
* `sed -i''` is invisible to the launching process and identical to `sed -i`. (Works on Linux only)
* `sed -i ''` works on Mac OS, but is invalid on Linux, since there's no space expected for the short argument, and `--in-place` does not exist on Mac OS.

Only a non-empty string not separated by space seems to be a acceptable to both variants.

Reply:
Now deleting the bak file immediately afterwards.